src/util-hash.c

```diff
@@ -208,6 +208,20 @@ void *HashTableLookup(HashTable *ht, void *data, uint16_t datalen)
     return NULL;
 }
 
+void HashTableIterate(HashTable *ht, void (*callback)(void *))
```

Comment:
For this use case we generally use a hash list table

Reply:
also: style: `CallbackFn`

Reply:
changed the style, left the iterator for now.